2020-07-11 11:02:30 base_spider.py [line:39] INFO : Request for http://www.xiladaili.com/gaoni/1/ returned <Response [200]>. Pause for 1.4397060896035656 seconds
2020-07-11 11:12:49 base_spider.py [line:39] INFO : Request for http://www.xiladaili.com/gaoni/1/ returned <Response [200]>. Pause for 2.2119632217635674 seconds
2020-07-11 11:16:41 base_spider.py [line:39] INFO : Request for https://www.kuaidaili.com/free/inha/1/ returned <Response [200]>. Pause for 2.8420609572354447 seconds
2020-07-11 11:16:43 base_spider.py [line:39] INFO : Request for http://www.xiladaili.com/gaoni/1/ returned <Response [200]>. Pause for 1.068643036930214 seconds
2020-07-11 11:17:26 mongo_pool.py [line:26] WARNING : 在数据库 proxies_pool.proxies 已存在代理 : 184.82.128.211:8080
2020-07-11 11:18:05 mongo_pool.py [line:24] INFO : 向数据库 proxies_pool.proxies 插入新代理 : 103.79.155.42:46977
2020-07-11 11:20:29 mongo_pool.py [line:24] INFO : 向数据库 proxies_pool.proxies 插入新代理 : 144.76.179.68:59991
2020-07-11 11:36:17 base_spider.py [line:39] INFO : Request for https://www.kuaidaili.com/free/inha/1/ returned <Response [200]>. Pause for 1.6976602591412389 seconds
2020-07-11 11:36:18 base_spider.py [line:39] INFO : Request for http://www.xiladaili.com/gaoni/1/ returned <Response [200]>. Pause for 2.5743099116739128 seconds
2020-07-11 11:36:49 run_spiders.py [line:43] ERROR : name 'https_nick_tyoe' is not defined
Traceback (most recent call last):
  File "/Users/l/Documents/Projects/ip_pool/core/proxy_spider/run_spiders.py", line 37, in __execute_one_spider_task
    proxy = check_proxy(proxy)
  File "/Users/l/Documents/Projects/ip_pool/core/proxy_validate/httpbin_validator.py", line 34, in check_proxy
    proxy.nick_type = https_nick_tyoe
NameError: name 'https_nick_tyoe' is not defined
2020-07-11 11:38:17 base_spider.py [line:39] INFO : Request for https://www.kuaidaili.com/free/inha/1/ returned <Response [200]>. Pause for 2.247494523219127 seconds
2020-07-11 11:38:18 base_spider.py [line:39] INFO : Request for http://www.xiladaili.com/gaoni/1/ returned <Response [200]>. Pause for 2.945578113979609 seconds
2020-07-11 11:38:47 mongo_pool.py [line:26] WARNING : 在数据库 proxies_pool.proxies 已存在代理 : 184.82.128.211:8080
